currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/MyDAOsFragment.kt

```diff
@@ -120,21 +120,35 @@ class MyDAOsFragment : BaseFragment(R.layout.fragment_my_daos) {
             "$TEST_NET_WALLET_NAME.wallet"
         )
 
+        val vWalletFileRegTest = File(
+            this.requireContext().applicationContext.filesDir,
+            "$REG_TEST_WALLET_NAME.wallet"
+        )
+
         val mainNetWalletExists = vWalletFileMainNet.exists()
         val testNetWalletExists = vWalletFileTestNet.exists()
-        val hasTwoWalletFiles = mainNetWalletExists && testNetWalletExists
-        val hasOneWalletFile = mainNetWalletExists xor testNetWalletExists
-        val hasNoWalletFiles = !mainNetWalletExists && !testNetWalletExists
-
-        if (hasTwoWalletFiles && !WalletManagerAndroid.isInitialized()) {
+        val regTestWalletExists = vWalletFileRegTest.exists()
+        val hasThreeWalletFiles = mainNetWalletExists && testNetWalletExists && regTestWalletExists
+        val hasTwoWalletFiles = (mainNetWalletExists && testNetWalletExists) xor (mainNetWalletExists && regTestWalletExists) xor (testNetWalletExists && regTestWalletExists)
```

Comment:
This code can be done so much easier:

```
val wallets = arrayOf(mainNetWalletExists, testNetWalletExists, regTestWalletExists)
val walletCount = wallets.count { true }

if (walletCount > 1) {
     Log.i("Coin", "$(walletCount) wallet files exist, navigation to choice screen.")
     navController.navigate(MyDAOsFragmentDirections.actionMyDAOsFragmentToDaoLoginChoice())
}

```

Then get rid of all the xors and conditions above